Doc/howto/functional.rst

```diff
@@ -608,8 +593,8 @@ generators:
   will also be called by Python's garbage collector when the generator is
   garbage-collected.
 
-  If you need to run cleanup code when a :exc:`GeneratorExit` occurs, I suggest
-  using a ``try: ... finally:`` suite instead of catching :exc:`GeneratorExit`.
+  If you need to run cleanup code when a :exc:`GeneratorExit` occurs, it's recommended
+  to use a ``try: ... finally:`` suite instead of catching :exc:`GeneratorExit`.
```

Comment:
I prefer the current wording, which is more concise, and more upfront about the fact that this is one person's opinion.

Reply:
I doubt I had a personal opinion about catching vs. try...finally, & vaguely think I got the suggestion from somewhere. Could it have been from python-dev discussion when the feature was being added? Or maybe it was just general concern about the complexity of dealing with the GeneratorExit; do you need to re-raise it once your cleanup is done?  Maybe this should say "it's simpler to use try...finally" instead of 'recommended'?  (And do 3.x async features change any of this?)